src/validator.js

```diff
@@ -0,0 +1,64 @@
+import * as fs from 'fs';
+import Xpi from  'xpi';
+
+import promisify from 'es6-promisify';
+import chalk from 'chalk';
+
+export var lstat = promisify(fs.lstat);
+
+
+export default class Validator {
+
+  constructor(config) {
+    this.config = config;
+    this.packagePath = config._[0];
+    this.xpi;
+    this.chalk = new chalk.constructor(
+      {enabled: !this.config.boring});
+  }
+
+  handleError(err) {
+    if (this.config.stack === true) {
```

Comment:
Fair enough. I’ll just make a note in the JSCS so we keep things consistent; I tend to opt for truthy but a failed test for style will be a great reminder for me ;-)

- tofumatt

On 30 September 2015 at 09:20:17, Stuart Colville (notifications@github.com) wrote:

In src/validator.js:

> +
> +export var lstat = promisify(fs.lstat);
> +
> +
> +export default class Validator {
> +
> +  constructor(config) {
> +    this.config = config;
> +    this.packagePath = config._[0];
> +    this.xpi;
> +    this.chalk = new chalk.constructor(
> +      {enabled: !this.config.boring});
> +  }
> +
> +  handleError(err) {
> +    if (this.config.stack === true) {
I'm not sure if that's a thing we particularly adhered to in payments, but I definitely tend to default using explicit comparisons over truthy/falsey ones.

—
Reply to this email directly or view it on GitHub.